frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -52,8 +55,14 @@ const StepOne = ({
       reValidateMode: 'onChange',
       defaultValues: fieldsObj,
       resolver: yupResolver(stepOne.Schema),
+      context: { voteType },
     });
 
+  const handleVoteType = (voteType) => {
+    setVoteType(voteType);
+    setValue('voteType', voteType);
+  };
+
```

Comment:
I think you can remove the state for holding voteType and passing it as a context to the hook by doing:
`const voteType = useWatch({ control, name: 'voteType' });`
then we just have the value in the form. (Below line 77)